radis/test/vaex_vs_pandas_performance.py

```diff
@@ -0,0 +1,118 @@
+import time
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from radis import calc_spectrum
+from radis.misc.progress_bar import ProgressBar
+
+
+def compare_vaex_pandas_time():
```

Comment:
I dislike that we need HITEMP H2O ; it's a very large database. 
Ok for your own performance tests, and to plot on the PR; but remember that these tests will be executed on every single commit during the next 5 years ! 
Let's be minimal. Use a database already downloaded in the other tests. 


Also, there is no assert statement anywhere here. So it's probably more a Benchmark than a Test. 
If confirmed, then add it in Radis/benchmarks : https://github.com/radis/radis-benchmark , where you can use whatever database you want as it's not ran on every new commit

(as well as in this PR text )

Reply:
These are not executed in every commit here is an example from last commit https://app.travis-ci.com/github/radis/radis/jobs/607697939. These functions were not executed in that as there name doesn't start or end with test so they are not detected by pytest and hence not executed.

The idea to use HITEMP CO2 was to have more clear idea about the performance of vaex for larger databases.

Reply:
You're right, I did not notice function names don't start with `test*` so we're good. You can resolve my comment. Maybe start with "benchmark_*"  and mention explicitely in the docstrings that they're long tests

```
.. warnings 
    This function will download the full H2O HITEMP database which requires > 10 (?) GB and many minutes if not hours of parsing. It is not executed by default in the test suite (name doesn't start with test_*)
```